client/src/App.css

```diff
@@ -8,26 +14,46 @@ body {
   overscroll-behavior: none;
 }
 
+/*
+  Avoid text overflows
+*/
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  overflow-wrap: break-word;
+}
+
 .container {
-  font-family: 'Roboto', sans-serif;
-  font-style: normal;
-  position: fixed;
+  display: flex;
   height: 100vh;
   width: 100%;
-  max-width: 1206;
-  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+
+  /* TODO-FIXME: Removed because it prevented children components' page scroll*/
+  /* position: fixed;  */
+  /*TODO-FIXME: Not displaing to the page?  */
+  /* max-width: 1206; */
 }
 .navbar {
   display: flex;
+  height: 10vh;
   width: 100%;
-  position: fixed;
-  border-bottom: 10 px solid grey;
-  height: 100px;
-  background: none;
-  top: 0;
-  z-index: 10;
-  border-bottom: 0.5px solid black;
   justify-content: space-around;
+  border-bottom: 0.5px solid black;
+  background-color: inherit;
+  /* background: url(./assets/images/pinkcrop1.jpg); */
+  /** Removed to allow for overall repositioning of the parent container layout with the navbar and other internal sibling components, i.e. <BusinessList/> or <AddBusiness/> */
```

Comment:
is this relevant anymore as parent container is fixed? please explain if so, thanks!